src/editor/parser/render/renderInlines/inlineMath.js

```diff
@@ -1,3 +1,5 @@
+import { snakeToCamel } from '../../../utils'
+
 export default function inlineMath (h, cursor, block, token, outerClass) {
-  return this['display_math'](h, cursor, block, token, outerClass)
+  return this[snakeToCamel('display_math')](h, cursor, block, token, outerClass)
```

Comment:
😆，I think we can direct write `displayMath`.

It's my fault!

`this.displayMath(...)`